editor/src/components/canvas/canvas-strategies/strategies/draw-to-insert-metastrategy.tsx

```diff
@@ -656,3 +667,11 @@ function runTargetStrategiesForFreshlyInsertedElementToResize(
 
   return foldAndApplyCommandsInner(editorState, [], resizeCommands, commandLifecycle).statePatches
 }
+
+export function getRootPath(startingMetadata: ElementInstanceMetadataMap): ElementPath | null {
```

Comment:
Please can you rename this function to `getStoryboardPath` and move it into `element-metadata-utils.ts`

Reply:
https://github.com/concrete-utopia/utopia/pull/3623/commits/53a9ce32850d54b16b0b2310e7e40895caac1bdb